src/ronny.js

```diff
@@ -34,8 +34,13 @@ bot.on('message', function(data) {
           dict.forEach(d => {
             if (d.keys.some(k => ~msg.indexOf(k))) {
               var message = d.message;
+              var params = default_params;
               if (typeof message === 'function')
                 message = message();
+              if (message.params) {
+                params = message.params;
```

Comment:
These should be merged with the default params so we don't have to specify all of them every time. Maybe there are helpers in the lodash package which consists of a ton of utility functions.